src/ProjectContext.php

```diff
@@ -84,12 +84,10 @@ public function locateFunctionSource(?string $functionSource): ?string
      */
     public function registerCloudStorageStreamWrapperIfPossible()
     {
-        if (class_exists(StreamWrapper::class)) {
-            if (!in_array('gs', stream_get_wrappers())) {
-                // Create a default GCS client and register the stream wrapper
-                $storage = new StorageClient();
-                StreamWrapper::register($storage);
-            }
+        if (class_exists(StreamWrapper::class) && !in_array('gs', stream_get_wrappers())) {
```

Comment:
It may be unnecessary but we use line-lengths of 80 characters in this (and all) of our repos. Also I think the other way is more readable. Can you revert this block?

Reply:
Thank you for the review. 

Sure. I've reverted this block.